include/k4a/k4a.hpp

```diff
@@ -1116,6 +1116,18 @@ class device
         return m_handle != nullptr;
     }
 
+    /** Returns the underlying k4a_device_t handle
+     *
+     * Note that this function does not increment the reference count on the k4a_device_t.
+     * The caller is responsible for incrementing the reference count on
+     * the k4a_device_t if the caller needs the k4a_device_t to outlive this C++ object.
+     * Otherwise, the k4a_device_t will be destroyed by this C++ object.
```

Comment:
Thank you for submitting this. The handle is not reference counted at all and there is unfortunately no way to  make sure the handle survives destruction. I would say something like: _** Note the k4a_device_t handle does not have a reference count will be destroyed when the C++ object is destroyed. The caller is responsible for ensuring the C++ object outlives this handle.**_

Reply:
Thanks! Yes, that's right!